src/atomate2/common/flows/gruneisen.py

```diff
@@ -0,0 +1,230 @@
+    generated for all the three structures (ground state, expanded and shrunk volume)
+    and accurate forces are computed for these structures. With the help of phonopy,
+    these forces are then converted into a dynamical matrix. This dynamical matrix of
+    three structures is then used as input phonopy Grueneisen api to compute Grueneisen
+    parameters are computed.
+
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    bulk_relax_maker: .ForceFieldRelaxMaker, .BaseAimsMaker, .BaseVaspMaker, or None
+        A maker to perform an initial tight relaxation on the bulk.
+    code: str
+        determines the dft or force field code.
+    const_vol_relax_maker: .ForceFieldRelaxMaker, .BaseAimsMaker,
+        .BaseVaspMaker, or None. A maker to perform a tight relaxation
+        on the expanded and shrunk structures at constant volume.
+    kpath_scheme: str
+        scheme to generate kpoints. Please be aware that
+        you can only use seekpath with any kind of cell
+        Otherwise, please use the standard primitive structure
+        Available schemes are:
+        "seekpath", "hinuma", "setyawan_curtarolo", "latimer_munro".
+        "seekpath" and "hinuma" are the same definition but
+        seekpath can be used with any kind of unit cell as
+        it relies on phonopy to handle the relationship
+        to the primitive cell and not pymatgen
+    mesh: tuple|float
+        Mesh numbers along a, b, c axes used for Grueneisen parameter computation.
+        Or float to indicate a kpoint density.
+    phonon_displacement_maker: .ForceFieldStaticMaker | .BaseVaspMaker | None
+    phonon_maker_kwargs: dict
+    perc_vol: float
+        Percent volume to shrink and expand ground state structure
+    compute_gruneisen_param_kwargs: dict
+        Keyword arguments passed to :obj:`compute_gruneisen_param`.
+    """
+
+    name: str = "Gruneisen"
+    bulk_relax_maker: ForceFieldRelaxMaker | BaseVaspMaker | BaseAimsMaker | None = None
+    code: str = None
+    const_vol_relax_maker: ForceFieldRelaxMaker | BaseVaspMaker | BaseAimsMaker = None
+    kpath_scheme: str = "seekpath"
+    phonon_displacement_maker: ForceFieldStaticMaker | BaseVaspMaker | None = None
+    phonon_static_maker: ForceFieldStaticMaker | BaseVaspMaker | None = None
+    phonon_maker_kwargs: dict = field(default_factory=dict)
+    perc_vol: float = 0.01
+    mesh: tuple | float = 7_000
+    compute_gruneisen_param_kwargs: dict = field(default_factory=dict)
+    symprec: float = 1e-4
+
+    def make(self, structure: Structure, prev_dir: str | Path | None = None) -> Flow:
+        """
+        Optimizes structure and runs phonon computations.
+
+        Phonon computations are run for ground state, expanded and shrunk
+        volume structures. Then, Grueneisen parameters are computed from
+        this three phonon runs.
+
+        Parameters
+        ----------
+        structure: Structure
```

Comment:
Missing doc string information, e.g. description of structure and also prev_dir.